packages/core/template/portfolio/contents/project1.md

```diff
@@ -0,0 +1,23 @@
+<frontmatter>
+  title: Project 1
+</frontmatter>
+
+<br>
+
+### Project 1
+
+<div class="card mb-3">
+  <h3 class="card-header">Name of project</h3>
+  <div class="card-body">
+    <h5 class="card-title">Short description of project</h5>
+  </div>
+  <pic src="../contents/assests/MarkbindExample.png" width="500" alt="project-display"/>
+  <div class="card-body">
+    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
+    {{typescript}}
+  </div>
+  <div class="card-body">
+    <a href="#" class="btn btn-primary">Card link</a>
```

Comment:
I was confused as to what this should lead to. maybe instead of "Card Link" you can go with "Some External Link Here" or similar so its more clear its sample text?